firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+KeyCommands.swift

```diff
@@ -145,9 +134,8 @@ extension BrowserViewController {
                                      method: .press,
                                      object: .keyCommand,
                                      extras: ["action": "new-tab"])
-        let isPrivate = tabManager.selectedTab?.isPrivate ?? false
-        openBlankNewTab(focusLocationField: true, isPrivate: isPrivate)
-        keyboardPressesHandler().reset()
+        store.dispatch(GeneralBrowserAction(windowUUID: windowUUID,
```

Comment:
Same as above: We should use the toolbar refactor feature flag here for the dispatch and otherwise execute the old code.